### DISCUSSION TITLE
M1Pro Macbook lost connection after installing

### DISCUSSION THREAD

**Discussion by ethanmiller16** [MonitorControl/MonitorControl#1011] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

Hi Guys,

I just installed monitorcontrol. After a couple of minutes and playing around with some settings I had a bit of a matrix looking screen flash up and then lost all feed from my macbook. My monitor now reads (HDMI 1 No signal). I've switched this out to my xbox and it works fine and restarted both monitor and mac and still no luck.

Any idea what could've caused this and how to fix it?

### Steps to reproduce

Install on m1pro macbook. 

### Expected behavior

Not sure

### Environment Information (please complete the following information)

```markdown
- macOS version:12.2.1 Monterey
- Mac model: 14 inch m1 pro
- MonitorControl version: 4.0
- Monitor(s): Gigabyte m32u
- Apple Silicon/M1 (yes or no): Yes
```

**Comment by waydabber** [comment]
Hi @ethanmiller16 - unfortunately the HDMI port of the new MBP does not work for proper control, you need to use USB-C.

If you connect your macbook to an other HDMI display (a TV for example), do you experience the same problem?

If you try to switch the resolution or enable/disable HDR, do you still have this problem? You should try switching to a non-native based unscaled one (you might have to use an app for that like [BetterDummy](https://github.com/waydabber/BetterDummy/releases/tag/v1.1.12-rc) - enable `Show additional resolution options` and switch to `Use resolution submenu` under Preferences/Resolutions) that _might_ reset things. Also you can delete `com.apple.windowserver.displays...` by following steps 3,4 and 6 from here: https://github.com/waydabber/BetterDummy/discussions/191#discussioncomment-1763550

If these don't help then some hardware level reset might be needed if processing the DDC command somehow messed up the HDMI converter chip inside the macbook (this is the component that is problematic on m1 macs with HDMI port). https://github.com/MonitorControl/MonitorControl/discussions/750

**Reply by godzfire** [reply]
I have a similar issue I am going to put a report in for but also just wanted to mention it here. Using the app on a Mac Mini 2018 connected via HDMI to HDMI port to an LG C1 OLED. If hardware and software dimming is on (which I use to get it as dark as possible), it will randomly disconnect from the TV and say "No input source". Reconnecting the cables does nothing, nor does putting the computer to sleep. Only a full shut down, wait a few seconds, and then rebooting will restore the connection.

I haven't been able to sadly use MonitorControl because of it, and it's really like to, because it gets the screen darker than the other dimmer apps out there (like Shade).

Here's the displays page for it in MC
<img width="730" alt="image" src="https://user-images.githubusercontent.com/2067820/179064574-6b36f909-9d2b-44d4-bad9-a63759f94043.png">

**Reply by ducquynhnguyeninfo** [reply]
> 

I have the issue on the MacBook pro m1 14inch with Dell U2417H too. While plugged in the HDMI port, only the color shade/gamma changes and the brightness on the monitor doesn't (I can tell by seeing the brightness in the monitor's built-in settings menu board). Connecting the screen through a Type-C hub didn't help in my case.

![IMG_5082](https://user-images.githubusercontent.com/7911292/179663523-4efe2c37-6f6d-4460-9daa-fa78614f22fc.jpeg)

I also did check on the Macbook pro-15-inch, mid-2015 on the same screen, everything worked so well.
